ACPs/75-acceptance-proofs/README.md

```diff
@@ -13,11 +13,11 @@ Introduces support for a proof of a block’s acceptance in consensus.
 
 ## Motivation
 
-Subnets use the [ProposerVM](https://github.com/ava-labs/avalanchego/blob/416fbdf1f783c40f21e7009a9f06d192e69ba9b5/vms/proposervm/README.md) to implement a soft leader mechanism. The ProposerVM determines the block producer schedule from a randomly shuffled validator set for that subnet at each block height. The ProposerVM wrapper specifies the P-Chain height where it was verified.
+Subnets are able to prove arbitrary events using warp messaging, but native support for proving block acceptance at the protocol layer enables more utility. One use case for acceptance proofs is to provide stronger fault isolation guarantees from the primary network to subnets.
```

Comment:
(No change necessary) It would be helpful to enumerate all of the intended use cases instead of just fault isolation. As it's currently phrased, this begs the question, what other use cases are there?